conftest.py

```diff
@@ -57,3 +58,27 @@ def small_client(small_cluster):
         client.wait_for_workers(10)
         client.restart()
         yield client
+
+
+@pytest.fixture(scope="session")
+def s3_region():
+    return "us-east-1"
+
+
+@pytest.fixture(scope="session")
+def s3_bucket():
+    return "dask-io"
+
+
+@pytest.fixture(scope="session")
+def s3_storage_options(s3_region):
+    return {"config_kwargs": {"region_name": s3_region}}
```

Comment:
If this is the only option we are providing, do we need one fixture to define the region and one to pass the region?

Reply:
`s3_region` is used by the `s3` fixture, but also in other fixtures where it's passed in the `storage_options`.

I guess we could drop `s3_storage_options` and just use `s3_region` in the multiple places where storage options are required, but there's more possibility for something to go wrong IMO.